testcases/kernel/security/stack_clash/stack_clash_child.c

```diff
@@ -0,0 +1,93 @@
+/* File from https://bugzilla.novell.com/show_bug.cgi?id=CVE-2017-1000364 */
+
+#include <sys/mman.h>
+#include <sys/resource.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <alloca.h>
+#include <signal.h>
+#include <stdlib.h>
+
+#define TST_NO_DEFAULT_MAIN
+#include "tst_test.h"
+
+#define PAGE_SHIFT              12
+#define PAGE_SIZE               (1UL << PAGE_SHIFT)
+#define PAGE_MASK		(~(PAGE_SIZE-1))
+#define SIGNAL_STACK_SIZE	(1UL<<20)
+#define MEGABYTE		(1<<20)			
+#define THRESHOLD		(250 * PAGE_SIZE)
```

Comment:
This would be problem if someone runs this on system booted with stack_guard_gap=X other than default (256).